doc/cabal-package.rst

```diff
@@ -801,18 +800,45 @@ Library
 
     Build information for libraries.
 
-    A package can have one default library, which is identified by omitting the
-    ``name`` argument.
+    A package can include zero or more library components, being an optional
+    'main' unnamed library (which is identified by having no ``name`` argument)
+    and/or zero or more 'subsidiary' or 'ancilliary' named libraries (referred
+    to as 'sublibraries', which are identified by their ``name`` argument). A
+    sublibrary cannot have the same name as its package.
```

Comment:
I'm happy to split into shorter sentences. My reason for referring to 'subsidiary' and 'ancillary' is to explain the (likely) origin of (and so give content to) the abbreviated term 'sublibrary' for named library components. I included both because some people preferred 'ancilliary' to 'subsidiary'. (Their meanings are very similar.)

Reply:
I'm currently wondering if "related library" is better and clearer than either. (I was originally in favor of "ancillary", but…
```
[26 01:45:36] <haskellbridge> <g​eekosaur> I considered "ancillary" but half expect people to look at me like I said "monad"…
```
(The defense rests.))

Reply:
Haha very prescient!